lib/simple_metric/metric.rb

```diff
@@ -0,0 +1,57 @@
+module SimpleMetric
+  class Metric < ActiveRecord::Base
+    MAX_SIZE = 1000
+
+    serialize :data_set
+    validates :key, presence: true, uniqueness: true
+
+    # @param [String] key.
+    # @param [Time] date.
+    # @param [Numeric] value.
+    #
+    # @return [true]
+    def self.add_data_point(key, date, value)
+      find_or_create_by(key: key).add_data_point(date, value)
+    end
+
+    # @example ['String', 'Time', 'Numeric']
+    # @param [String] key.
+    # @param [Time] date.
+    # @param [Numeric] value.
+    def self.add_data_points(*points)
+      points.each do |point|
+        find_or_create_by(key: point[0]).add_data_point(point[1], point[2])
```

Comment:
Why can't we just reuse add_data_point?

Reply:
We can reuse the existing method. But it has a singular name "add_data_point". So as not to mislead. In my opinion, it is better to add the "add_data_points" method.

Reply:
I mean we can reuse `add_data_point` internally inside `add_data_points`